src/grt/src/fastroute/include/FastRoute.h

```diff
@@ -99,7 +99,7 @@ class FastRouteCore
   ~FastRouteCore();
 
   void clear();
-  void saveCongestion();
+  void saveCongestion(const int& iter = -1);
```

Comment:
Just use int - pass by value is better for primitive types.

Reply:
use std::option instead of overloading int? self documenting...